module/VuFind/tests/unit-tests/src/VuFindTest/ILS/ConnectionTest.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+/**
+ * Connection test
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFindTest\ILS;
+
+use VuFind\Config\Config;
+use VuFind\ILS\Connection;
+
+/**
+ * Connnection test
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Jaro Ravila <jaro.ravila@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class ConnectionTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Connection object
+     *
+     * @var Connection
+     */
+    protected $connection;
+
+    /**
+     * Standard setup method.
+     *
+     * @return void
+     */
+    public function setUp(): void
+    {
+        $config = new Config(['driver' => 'Demo']);
+        $driverManager = $this->getMockBuilder(\VuFind\ILS\Driver\PluginManager::class)
+            ->disableOriginalConstructor()->getMock();
+        $driverManager->method('has')->willReturn('Demo');
+        $configReader = $this->getMockBuilder(\VuFind\Config\PluginManager::class)
+            ->disableOriginalConstructor()->getMock();
+        $this->connection = new Connection(
+            $config,
+            $driverManager,
+            $configReader
+        );
+    }
+
+    /**
+     * Set TimedBlocks driver configuration
+     *
+     * @param array $timedBlocks timed blocks as defined in driver.ini
```

Comment:
Should we use the actual filename here?

```suggestion
     * @param array $timedBlocks timed blocks as defined in Demo.ini
```